packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md

```diff
@@ -1,3 +1 @@
-A Dropdown is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button. They are used to simplify the design and make a choice within the UI. When closed, only the selected item is visible. When users click the drop-down button, all the options become visible. To change the value, users open the list and click another value or use the arrow keys (up and down) to select a new value.
-
-**Note:** Dropdown content automatically renders in a Panel on screens under 480px wide. To disable this behavior set `<Dropdown responsiveMode={ResponsiveMode.Large} ... />`
+A drop-down menu (`Dropdown`) is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button.
```

Comment:
```suggestion
A drop-down menu (`Dropdown`) is a list in which the selected item is always visible, while the non-selected items are visible on demand by clicking a drop-down button.
```

Reply:
I didn't go with your exact suggestion, because "non-selected" felt a bit too technical. So, I edited this to, "...the selected item is always visible while other items are visible on demand by selecting a dropdown button."